src/CMakeLists.txt

```diff
@@ -33,7 +34,10 @@ target_compile_definitions(mc_rbdyn_urdf PRIVATE "-DMCRBDYNURDF_BUILDING")
 set(INSTALL_PATH include/mc_rbdyn_urdf)
 install(FILES ${HEADERS} DESTINATION ${INSTALL_PATH})
 install(TARGETS mc_rbdyn_urdf
-  RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib
+EXPORT "${TARGETS_EXPORT_NAME}"
+LIBRARY DESTINATION "lib"
+ARCHIVE DESTINATION "lib"
+RUNTIME DESTINATION "bin"
+INCLUDES DESTINATION "${INCLUDE_INSTALL_DIR}"
 )
+install(FILES ${HEADERS} DESTINATION ${INCLUDE_INSTALL_DESTINATION})
```

Comment:
This is redundant with `install(FILES ${HEADERS} DESTINATION ${INSTALL_PATH})`

Reply:
This line has also been removed.